src/components/CallView/shared/LocalVideo.vue

```diff
@@ -314,6 +314,10 @@ export default {
 			this.$store.dispatch('selectedVideoPeerId', null)
 			this.$store.dispatch('stopPresentation')
 		},
+
+		switchScreenToId(id) {
+			this.$emit('switch-screen-to-id', id)
```

Comment:
some day we'll move this to the store so we don't have to pass it around everywhere :city_sunset: